packages/crayons-core/src/components/select/select.tsx

```diff
@@ -23,6 +23,24 @@ import {
 } from '../../utils/types';
 
 import { i18n } from '../../global/Translation';
+import {
+  changeEmittable,
+  clearInput,
+  closeDropdown,
+  expandHandler,
+  focusHandler,
+  fwChangeHandler,
+  hideListHandler,
+  initializeComponent,
+  innerClickHandler,
+  innerFocusHandler,
+  innerOnBlur,
+  isValueEqual,
+  openDropdown,
+  optionsWatcher,
+  renderInput,
+  valueExists,
+} from '../../utils/select-utils';
```

Comment:
Now that there is no email select and select and have been merged, is the utils still needed ?

Reply:
This is not needed @kaushal-freshworks. Hence, I have closed this PR.

CC @arvindanta